packages/openbridge-webcomponents/src/navigation-instruments/readout-input/readout-input.css

```
@@ -0,0 +1,747 @@
+    display: flex;
+    justify-items: end;
+    align-items: center;
+
+    .value-content-container {
+      display: flex;
+    }
+
+    .degree {
+      grid-column: 2;
+      @mixin font-instrument-value-m-active;
+      display: block;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)